rust/signer/src/metadata.rs

```diff
@@ -0,0 +1,134 @@
+use hex;
```

Comment:
Whats the advantage of rolling up one-off metadata processing vs using @polkadot/metadata?

Reply:
noticeably faster